### PR TITLE
[curl] update to 7.55.0

### PR DESCRIPTION
**Pull request by codicodi**

### CODE REVIEW

File: ports/curl/portfile.cmake [hunk 1]
```diff
@@ -14,40 +14,36 @@ vcpkg_apply_patches(
...
+
+vcpkg_find_acquire_program(PERL)
+get_filename_component(PERL_PATH ${PERL} DIRECTORY)
+set(ENV{PATH} "$ENV{PATH};${PERL_PATH}")
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA
+    OPTIONS
+        ${UWP_OPTIONS}
```
Comment (alexkaratarakis) [inline ports/curl/portfile.cmake]: Are the UWP_OPTIONS always passed in?

Comment (codicodi) [inline ports/curl/portfile.cmake]: Huh, they shouldn't be. Fixed

**Comment by alexkaratarakis** [comment]
Thanks!

**Comment by martin-s** [comment]
I would rather see the cmake script been fixed as dropping the d-postfix in the lib. It's common practice to add some debug info to the library name.

**Pull request merged**